rabbitmq/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use the Bitnami RabbitMQ image as the base
-FROM bitnami/rabbitmq:3.12
+FROM rabbitmq:3.13.1
```

Comment:
not sure I'm reading this correctly, but if it's the case we're not using the Bitnami library, can we remove the comment on the previous line?

Reply:
yes, good catch, this was a mistake and shouldn't have been included in the pr.  it was just something i was playing with after seeing the debian vulnerabilities pop back up for the nth time.